xml/System.Threading/ManualResetEventSlim.xml

```diff
@@ -56,7 +56,9 @@
 ## Remarks  
  You can use this class for better performance than <xref:System.Threading.ManualResetEvent> when wait times are expected to be very short, and when the event does not cross a process boundary. <xref:System.Threading.ManualResetEventSlim> uses busy spinning for a short time while it waits for the event to become signaled. When wait times are short, spinning can be much less expensive than waiting by using wait handles. However, if the event does not become signaled within a certain period of time, <xref:System.Threading.ManualResetEventSlim> resorts to a regular event handle wait.  
   
-   
+> [!NOTE]
+>  In .NET Core and .NET 5+, the default spin-waiting duration is very short, in the order of 10s of microseconds depending on platform and processor. If wait times are expected to be much longer than that, this class can still be used instead of <xref:System.Threading.ManualResetEvent> (perhaps configured with less or no spin-waiting), but the performance benefit would likely be only marginal.  
```

Comment:
```suggestion
> In .NET Core and .NET 5+, the default spin-waiting duration is short: on the order of 10s of microseconds, depending on platform and processor. If you expect wait times to be much longer than that, you can still use this class instead of <xref:System.Threading.ManualResetEvent> (perhaps configured with less or no spin-waiting). However, the performance benefit would likely be only marginal.  
```